modules/simple_forms_api/lib/tasks/resubmit_forms_by_uuid.rake

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_intake/service'
+require 'simple_forms_api_submission/metadata_validator'
+
+# Invoke this as follows (tested on ZShell):
+#   rails "simple_forms_api:resubmit_forms_by_uuid[abc-123 def-456]"
+namespace :simple_forms_api do
+  task :resubmit_forms_by_uuid, [:benefits_intake_uuids] => :environment do |_, args|
+    benefits_intake_uuids = args[:benefits_intake_uuids].split
+    benefits_intake_uuids.each do |benefits_intake_uuid|
+      # Get the original submission
+      form_submission = FormSubmission.find_by(benefits_intake_uuid:)
+      next unless form_submission
+
+      # Re-generate the form PDF
+      form_id = SimpleFormsApi::V1::UploadsController::FORM_NUMBER_MAP[form_submission.form_type]
+      form = "SimpleFormsApi::#{form_id.titleize.gsub(' ', '')}".constantize.new(form_submission.form_data)
+      filler = SimpleFormsApi::PdfFiller.new(form_number: form_id, form:)
+      file_path = filler.generate(override_timestamp: form_submission.created_at)
```

Comment:
This `override_timestamp` is where the magic happens. The intent here is to allow us to specify the timestamp that gets stamped onto the PDF, rather than defaulting to `Time.now` (or something like that).